poly/src/polynomial/univariate/sparse.rs

```diff
@@ -2,10 +2,11 @@
 use crate::univariate::{DenseOrSparsePolynomial, DensePolynomial};
 use crate::{EvaluationDomain, Evaluations, UVPolynomial};
 use ark_ff::{FftField, Field};
+use ark_serialize::*;
 use ark_std::{collections::BTreeMap, fmt, vec::Vec};
 
 /// Stores a sparse polynomial in coefficient form.
-#[derive(Clone, PartialEq, Eq, Hash, Default)]
+#[derive(Clone, PartialEq, Eq, Hash, Default, CanonicalSerialize, CanonicalDeserialize)]
```

Comment:
Does this need to be serializable?

Reply:
In order for the `marlin_pc::Randomness` struct to be serializable, you need `UVPolynomial: CanonicalSerialize + CanonicalDeserialize` so ya.

Reply:
but `SparsePolynomial` doesn't implement `UVPolynomial`, no? Or am I not finding the right impl?

(I don't see `DenseOrSparsePolynomial` in the diff, so I assume you're not deriving it for that)

Reply:
Oh good point haha I forgot about that - will adjust.